lib/straight/gateway.rb

```diff
@@ -57,6 +59,8 @@ def self.included(base)
       return { period: period, iteration_index: iteration_index }
     end
 
+    TESTNET_ADAPTER = Straight::Blockchain::MyceliumAdapter.testnet_adapter
```

Comment:
https://github.com/snitko/straight/blob/master/lib/straight/blockchain_adapters/mycelium_adapter.rb#L13
that's higly unreliable, it can mutated into mainnet adapter